.env.sample

```diff
@@ -12,5 +12,6 @@ WARNINGS_THRESH=x
 PAPERTRAIL_HOST=x
 PAPERTRAIL_PORT=x
 TWITTER_BEARER_TOKEN=x
+GUILD_ID=852656268090802263=x
```

Comment:
Nix

Reply:
This is used for local development of slash commands. Since slash commands need to be deployed, as developer will enter the guild ID they are developing in and lion will deploy commands in said guild so they can test them,

Reply:
why not loop over the guilds like i do in https://github.com/cs-discord-at-ucf/lion/pull/738/files#diff-13c50b4cc67456d5ba9f4608265a1f770f856e4ab28225bf35ba2e4bfec7bea6